Pushing v6.32.2 to Microsoft store fails at missing "Endless Key.mp4"

We have a good demo video **Endless Key.mp4** on Microsoft store since **Submission 17**.
However, pushing Alpha Test Flight to MS store becomes failed after "Submission 17" and shows error:
```
Write-Error: C:\Users\runneradmin\Documents\PowerShell\Modules\StoreBroker\1.21.0\StoreBroker\PackageTool.ps1:3760
Line |
3760 |          Write-Log -Exception $_ -Level Error
     |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | 2023-08-10 03:46:04 : runneradmin :  Write-Log:
     | C:\Users\runneradmin\Documents\PowerShell\Modules\StoreBroker\1.21.0\StoreBroker\PackageTool.ps1:1279 Line |
     | 1279 |          Write-Log -Message $output -Level Error      |          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
     | | 2023-08-10 03:46:04 : runneradmin : Could not find media file [Endless Key.mp4] in any subdirectory of      |
     | [D:\a\endless-key-app\endless-key-app\MSAppPayload\PDPs\en-us].

Error: Process completed with exit code 1.
```
https://github.com/endlessm/endless-key-app/actions/runs/5816622993/job/15770191811

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -469,6 +469,19 @@ jobs:
             echo NULL > $dummy_screenshot
           }
 
+          echo "Going to create dummy trailers & their images for Application Payload creation:"
+          $xmlDoc.ProductDescription.Trailers.Trailer | ForEach-Object {
+            $dummy_trailer = "MSAppPayload/PDPs/en-us/" + $_.FileName
+            echo $dummy_trailer
+            echo NULL > $dummy_trailer
+
+            $_.Images.Image | ForEach-Object {
+              $dummy_image = "MSAppPayload/PDPs/en-us/" + $_.FileName
+              echo $dummy_image
+              echo NULL > $dummy_image
+            }
+          }
+
           New-SubmissionPackage `
             -ConfigPath "MSAppPayload/SBConfig.json" `
             -PDPRootPath "MSAppPayload/PDPs" `

